admin_guide/cluster_capacity.adoc

```diff
@@ -0,0 +1,251 @@
+[[admin-guide-cluster-capacity]]
+= Analyzing Cluster Capacity
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+As a cluster administrator, you can use the cluster capacity tool to view the
+number of pods that can be scheduled in order to increase the current resources
+before they become exhausted, and to ensure any future pods can be scheduled.
+This capacity comes from an individual node host in a cluster, and includes CPU,
+memory, disk space, and others.
+
+The cluster capacity tool simulates a sequence of scheduling decisions to
+determine how many instances of an input pod can be scheduled on the cluster
+before it is exhausted of resources in order to provide a more accurate
+estimation.
+
+[NOTE]
+====
+The remaining allocatable capacity is a rough estimation, because it does not
+count all of the resources being distributed among nodes. It analyzes only the
+remaining resources and estimates the available capacity that is still
+consumable in terms of a number of instances of a pod with given requirements
+that can be scheduled in a cluster.
+
+Also, pods might only have scheduling support on particular sets of nodes based
+on its selection and affinity criteria. As a result, the estimation of which
+remaining pods a cluster can schedule can be difficult.
+====
+
+The cluster capacity analysis tool can be run either as a stand-alone utility
+from the command line, or as a job in a pod inside an {product-title} cluster in
+order for the pod to be run multiple times.
+
+[[cluster-capacity-running-analysis]]
+== Running Cluster Capacity Analysis on the Command Line
+
+To run the tool on the command line:
+
+----
+$ cluster-capacity --kubeconfig <path-to-kubeconfig> \
+    --podspec <path-to-pod-spec>
+----
+
+The `--kubeconfig` option indicates your Kubernetes configuration file, and the
+`--podspec` option indicates a sample pod specification file, which the tool
+uses for estimating resource usage. The `podspec` specifies its resource
+requirements as `limits` or `requests`. The cluster capacity tool takes the
+pod's resource requirements into account for its estimation analysis.
+
+An example of the pod specification input is:
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: small-pod
+  labels:
+    app: guestbook
+    tier: frontend
+spec:
+  containers:
+  - name: php-redis
+    image: gcr.io/google-samples/gb-frontend:v4
+    imagePullPolicy: Always
+    resources:
+      limits:
+        cpu: 150m
+        memory: 100Mi
+      requests:
+        cpu: 150m
+        memory: 100Mi
+----
+
+The `--verbose` option can also be provided as part
```

Comment:
Maybe change to 'Use the `--verbose` option to get a...` OR 'You can also add `--verbose` option to the previous command to get a detailed ..."  s/output/get